test/run

```diff
@@ -2,10 +2,12 @@
 
 set -eu
 
-# run static code checks like pyflakes and pep8
-PYEXEFILES="$(git grep -lI '^#!.*python') $(git ls-files "*.py")"
-flake8 $PYEXEFILES
+if [ -z "${SKIP_STATIC_CHECK:-}" ]; then
+	# run static code checks like pyflakes and pep8
```

Comment:
Please update the comment while you are at it, we don't run either of these two any more.